Gems/EMotionFX/Code/Source/Integration/Editor/Components/EditorActorComponent.cpp

```diff
@@ -544,8 +544,57 @@ namespace EMotionFX
 
         void EditorActorComponent::OnAssetReloaded(AZ::Data::Asset<AZ::Data::AssetData> asset)
```

Comment:
is "asset" now an unreferenced argument/variable?

Reply:
Interesting the compiler didn't complain about that... Ill remove it